app/api/schema/tickets.py

```diff
@@ -67,6 +67,12 @@ def validate_quantity(self, data):
                 raise UnprocessableEntity({'pointer': '/data/attributes/quantity'},
                                           "quantity should be greater than or equal to max-order")
 
+    @validates_schema
+    def validate_price(self, data):
+        if data['type'] == 'paid' and ('price' not in data or not isinstance(data['price'], float)):
```

Comment:
No need to check for the data type of `data['price']`. Also, make sure that a paid ticket has a `price > 0`.

Reply:
ok! adding the check